files/default/logrotate_awslogs

```diff
@@ -0,0 +1,15 @@
+# DO NOT EDIT! This file is maintained by Chef
+# ALL EDITS WILL BE LOST WHEN CHEF RUNS
+
+/var/log/awslogs.log {
```

Comment:
How much effort would it be to make this a template so we can tweak some of the variables later if we wanted? I was chatting with Joe to make sure this change wouldn't impact some stuff he was doing that also use `chef-cwlogs` and he made this suggestion...seems like a good idea if it's relatively easy to implement.

Reply:
That is a simple change to make.  I chose to make it a file because I wanted it to be as uniform as possible without overriding the logrotate unless absolutely necessarily (like on ubuntu 12 or less where this change is needed).

If we were to make this a template and change any of the fields beyond the default, we would want to override it everywhere to make all OS's operate the same way.  I'm not a fan of doing this because it means that we take ownership of this file everywhere from now on, vs. just taking ownership of it on old versions of Ubuntu.